controllers/discovery/searchUser.js

```diff
@@ -56,6 +69,7 @@ const SearchUser = async (req, res) => {
         replacements: {
           likeQuery: `%${q}%`,
           userId,
+          topicIds,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the same issue of potential SQL injection with `topicIds`. Please sanitize or parameterize your inputs.
```diff
- topicIds,
+ [sequelize.escape(topicIds)],
```
Please note that the exact method for escaping or parameterizing inputs may vary depending on the specific SQL library you're using. The above examples are just illustrative.


<!-- This is an auto-generated comment by OSS CodeRabbit -->